finality-aleph/src/tcp_network.rs

```diff
@@ -186,25 +168,97 @@ impl TcpAddressingInformation {
             peer_id,
         })
     }
+
+    fn peer_id(&self) -> AuthorityId {
+        self.peer_id.clone()
+    }
+}
+
+/// A representation of TCP addressing information with an associated peer ID, self-signed.
+#[derive(Debug, Hash, Encode, Decode, Clone, PartialEq, Eq)]
+pub struct SignedTcpAddressingInformation {
```

Comment:
On one hand I get why such name, but on the other this part `try_from(legacy: Vec<LegacyTcpMultiaddress>)` makes this name senseless